Build failure: python39Packages.geoip2

### Steps To Reproduce

Steps to reproduce the behavior:
1. `nix-shell -p python39Packages.geoip2`

### Build log

```
this derivation will be built:
  /nix/store/x4qi0gqrfw2sjzss8g73xf77z7kqscng-python3.9-geoip2-4.6.0.drv
building '/nix/store/x4qi0gqrfw2sjzss8g73xf77z7kqscng-python3.9-geoip2-4.6.0.drv'...
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing setuptools-build-hook
Using setuptoolsBuildPhase
Using setuptoolsShellHook
Sourcing pip-install-hook
Using pipInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing setuptools-check-hook
Using setuptoolsCheckPhase
Sourcing pytest-check-hook
Using pytestCheckPhase
Removing setuptoolsCheckPhase
unpacking sources
unpacking source archive /nix/store/2f8067zznhbrj2bkhbbp2yfcv7mlkj8h-geoip2-4.6.0.tar.gz
source root is geoip2-4.6.0
setting SOURCE_DATE_EPOCH to timestamp 1655828327 of file geoip2-4.6.0/setup.cfg
patching sources
configuring
no configure script, doing nothing
building
Executing setuptoolsBuildPhase
running bdist_wheel
The [wheel] section is deprecated. Use [bdist_wheel] instead.
running build
running build_py
creating build
creating build/lib
creating build/lib/geoip2
copying geoip2/webservice.py -> build/lib/geoip2
copying geoip2/database.py -> build/lib/geoip2
copying geoip2/models.py -> build/lib/geoip2
copying geoip2/records.py -> build/lib/geoip2
copying geoip2/errors.py -> build/lib/geoip2
copying geoip2/types.py -> build/lib/geoip2
copying geoip2/__init__.py -> build/lib/geoip2
copying geoip2/mixins.py -> build/lib/geoip2
running egg_info
writing geoip2.egg-info/PKG-INFO
writing dependency_links to geoip2.egg-info/dependency_links.txt
writing requirements to geoip2.egg-info/requires.txt
writing top-level names to geoip2.egg-info/top_level.txt
reading manifest file 'geoip2.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
adding license file 'LICENSE'
writing manifest file 'geoip2.egg-info/SOURCES.txt'
copying geoip2/py.typed -> build/lib/geoip2
/nix/store/i9d5jnha7fh88a6zc6gd8knqcdm7k4yp-python3.9-setuptools-67.4.0/lib/python3.9/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
installing to build/bdist.linux-x86_64/wheel
running install
running install_lib
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/wheel
creating build/bdist.linux-x86_64/wheel/geoip2
copying build/lib/geoip2/webservice.py -> build/bdist.linux-x86_64/wheel/geoip2
copying build/lib/geoip2/database.py -> build/bdist.linux-x86_64/wheel/geoip2
copying build/lib/geoip2/py.typed -> build/bdist.linux-x86_64/wheel/geoip2
copying build/lib/geoip2/models.py -> build/bdist.linux-x86_64/wheel/geoip2
copying build/lib/geoip2/records.py -> build/bdist.linux-x86_64/wheel/geoip2
copying build/lib/geoip2/errors.py -> build/bdist.linux-x86_64/wheel/geoip2
copying build/lib/geoip2/types.py -> build/bdist.linux-x86_64/wheel/geoip2
copying build/lib/geoip2/__init__.py -> build/bdist.linux-x86_64/wheel/geoip2
copying build/lib/geoip2/mixins.py -> build/bdist.linux-x86_64/wheel/geoip2
running install_egg_info
Copying geoip2.egg-info to build/bdist.linux-x86_64/wheel/geoip2-4.6.0-py3.9.egg-info
running install_scripts
creating build/bdist.linux-x86_64/wheel/geoip2-4.6.0.dist-info/WHEEL
creating 'dist/geoip2-4.6.0-py2.py3-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
adding 'geoip2/__init__.py'
adding 'geoip2/database.py'
adding 'geoip2/errors.py'
adding 'geoip2/mixins.py'
adding 'geoip2/models.py'
adding 'geoip2/py.typed'
adding 'geoip2/records.py'
adding 'geoip2/types.py'
adding 'geoip2/webservice.py'
adding 'geoip2-4.6.0.dist-info/LICENSE'
adding 'geoip2-4.6.0.dist-info/METADATA'
adding 'geoip2-4.6.0.dist-info/WHEEL'
adding 'geoip2-4.6.0.dist-info/top_level.txt'
adding 'geoip2-4.6.0.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
Finished executing setuptoolsBuildPhase
installing
Executing pipInstallPhase
/build/geoip2-4.6.0/dist /build/geoip2-4.6.0
Processing ./geoip2-4.6.0-py2.py3-none-any.whl
Requirement already satisfied: aiohttp<4.0.0,>=3.6.2 in /nix/store/jwn9lpldrjb53w2538rg6211aad2ldkk-python3.9-aiohttp-3.8.4/lib/python3.9/site-packages (from geoip2==4.6.0) (3.8.4)
Requirement already satisfied: requests<3.0.0,>=2.24.0 in /nix/store/1np4411ilpk5nx0f07hmh0yw13bfidpm-python3.9-requests-2.28.2/lib/python3.9/site-packages (from geoip2==4.6.0) (2.28.2)
Requirement already satisfied: urllib3<2.0.0,>=1.25.2 in /nix/store/6l9zm2pls5lxi8nglz0absfhgf3w5658-python3.9-urllib3-1.26.14/lib/python3.9/site-packages (from geoip2==4.6.0) (1.26.14)
Requirement already satisfied: maxminddb<3.0.0,>=2.2.0 in /nix/store/h0z7vzcbjzw8wp4xlh85c1xrn45s4iz3-python3.9-maxminddb-2.2.0/lib/python3.9/site-packages (from geoip2==4.6.0) (2.2.0)
Requirement already satisfied: charset-normalizer<4.0,>=2.0 in /nix/store/71r3b26pmg77y8j5wqlnqw4wxay71x3c-python3.9-charset-normalizer-3.0.1/lib/python3.9/site-packages (from aiohttp<4.0.0,>=3.6.2->geoip2==4.6.0) (3.0.1)
Requirement already satisfied: yarl<2.0,>=1.0 in /nix/store/vj2azap4grl62kq3al12xmnhi69kzcbc-python3.9-yarl-1.8.2/lib/python3.9/site-packages (from aiohttp<4.0.0,>=3.6.2->geoip2==4.6.0) (1.8.2)
Requirement already satisfied: multidict<7.0,>=4.5 in /nix/store/7r1r54spqsgwq39ydvfm1jlp9w4flpq3-python3.9-multidict-6.0.4/lib/python3.9/site-packages (from aiohttp<4.0.0,>=3.6.2->geoip2==4.6.0) (6.0.4)
Requirement already satisfied: frozenlist>=1.1.1 in /nix/store/cyc05nml5hqfs5ciqcsrkdxpvdfbll2r-python3.9-frozenlist-1.3.3/lib/python3.9/site-packages (from aiohttp<4.0.0,>=3.6.2->geoip2==4.6.0) (1.3.3)
Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /nix/store/53yirnlqr96ias5k2pqbvgy9q4qpj675-python3.9-async-timeout-4.0.2/lib/python3.9/site-packages (from aiohttp<4.0.0,>=3.6.2->geoip2==4.6.0) (4.0.2)
Requirement already satisfied: aiosignal>=1.1.2 in /nix/store/9594njwcjw61gjhbbz9gkrfh7d74jidx-python3.9-aiosignal-1.3.1/lib/python3.9/site-packages (from aiohttp<4.0.0,>=3.6.2->geoip2==4.6.0) (1.3.1)
Requirement already satisfied: attrs>=17.3.0 in /nix/store/7fk6pqaa8hpkw7naszjdsxhlbwismnhw-python3.9-attrs-22.2.0/lib/python3.9/site-packages (from aiohttp<4.0.0,>=3.6.2->geoip2==4.6.0) (22.2.0)
Requirement already satisfied: idna<4,>=2.5 in /nix/store/3hpbmhv79ga0ld8bg8bxmgqd5s98wglb-python3.9-idna-3.4/lib/python3.9/site-packages (from requests<3.0.0,>=2.24.0->geoip2==4.6.0) (3.4)
Requirement already satisfied: certifi>=2017.4.17 in /nix/store/vlpgz8vnaahgzx8vhhrywzywkgddvi28-python3.9-certifi-2022.12.07/lib/python3.9/site-packages (from requests<3.0.0,>=2.24.0->geoip2==4.6.0) (2022.12.7)
Installing collected packages: geoip2
Successfully installed geoip2-4.6.0
/build/geoip2-4.6.0
Finished executing pipInstallPhase
pythonOutputDistPhase
Executing pythonOutputDistPhase
Finished executing pythonOutputDistPhase
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/44qdq0airxqwlj5b7c8z566yn03l3k12-python3.9-geoip2-4.6.0
checking for references to /build/ in /nix/store/44qdq0airxqwlj5b7c8z566yn03l3k12-python3.9-geoip2-4.6.0...
patching script interpreter paths in /nix/store/44qdq0airxqwlj5b7c8z566yn03l3k12-python3.9-geoip2-4.6.0
stripping (with command strip and flags -S -p) in  /nix/store/44qdq0airxqwlj5b7c8z566yn03l3k12-python3.9-geoip2-4.6.0/lib
shrinking RPATHs of ELF executables and libraries in /nix/store/6whj4q9dprd1dak625xfcjnka8616f8r-python3.9-geoip2-4.6.0-dist
checking for references to /build/ in /nix/store/6whj4q9dprd1dak625xfcjnka8616f8r-python3.9-geoip2-4.6.0-dist...
patching script interpreter paths in /nix/store/6whj4q9dprd1dak625xfcjnka8616f8r-python3.9-geoip2-4.6.0-dist
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
running install tests
no Makefile or custom installCheckPhase, doing nothing
pythonCatchConflictsPhase
pythonRemoveBinBytecodePhase
pythonImportsCheckPhase
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: geoip2
pytestCheckPhase
Executing pytestCheckPhase
============================= test session starts ==============================
platform linux -- Python 3.9.16, pytest-7.2.1, pluggy-1.0.0
rootdir: /build/geoip2-4.6.0
plugins: requests-mock-1.10.0
collected 151 items

tests/database_test.py ................................................. [ 32%]
.........................................                                [ 59%]
tests/models_test.py .........                                           [ 65%]
tests/webservice_test.py .....................F......................... [ 96%]
F....                                                                    [100%]

=================================== FAILURES ===================================
___________________________ TestClient.test_request ____________________________

self = <tests.webservice_test.TestClient testMethod=test_request>

    @httprettified
    def test_request(self):
        httpretty.register_uri(
            httpretty.GET,
            self.base_uri + "country/" + "1.2.3.4",
            body=json.dumps(self.country),
            status=200,
            content_type=self._content_type("country"),
        )
        self.run_client(self.client.country("1.2.3.4"))
        request = httpretty.last_request

        self.assertEqual(
            request.path, "/geoip/v2.1/country/1.2.3.4", "correct URI is used"
        )
        self.assertEqual(
>           request.headers["Accept"], "application/json", "correct Accept header"
        )
E       KeyError: 'Accept'

tests/webservice_test.py:303: KeyError
_________________________ TestAsyncClient.test_request _________________________

self = <tests.webservice_test.TestAsyncClient testMethod=test_request>

    @httprettified
    def test_request(self):
        httpretty.register_uri(
            httpretty.GET,
            self.base_uri + "country/" + "1.2.3.4",
            body=json.dumps(self.country),
            status=200,
            content_type=self._content_type("country"),
        )
        self.run_client(self.client.country("1.2.3.4"))
        request = httpretty.last_request

        self.assertEqual(
            request.path, "/geoip/v2.1/country/1.2.3.4", "correct URI is used"
        )
        self.assertEqual(
>           request.headers["Accept"], "application/json", "correct Accept header"
        )
E       KeyError: 'Accept'

tests/webservice_test.py:303: KeyError
=========================== short test summary info ============================
FAILED tests/webservice_test.py::TestClient::test_request - KeyError: 'Accept'
FAILED tests/webservice_test.py::TestAsyncClient::test_request - KeyError: 'Accept'
======================== 2 failed, 149 passed in 0.39s =========================
/nix/store/pgf87rgkzdbqhhsddpligy8xiwar63w8-stdenv-linux/setup: line 1594: pop_var_context: head of shell_variables not a function context
error: builder for '/nix/store/x4qi0gqrfw2sjzss8g73xf77z7kqscng-python3.9-geoip2-4.6.0.drv' failed with exit code 1;
       last 10 log lines:
       > >           request.headers["Accept"], "application/json", "correct Accept header"
       >         )
       > E       KeyError: 'Accept'
       >
       > tests/webservice_test.py:303: KeyError
       > =========================== short test summary info ============================
       > FAILED tests/webservice_test.py::TestClient::test_request - KeyError: 'Accept'
       > FAILED tests/webservice_test.py::TestAsyncClient::test_request - KeyError: 'Accept'
       > ======================== 2 failed, 149 passed in 0.39s =========================
       > /nix/store/pgf87rgkzdbqhhsddpligy8xiwar63w8-stdenv-linux/setup: line 1594: pop_var_context: head of shell_variables not a function context
       For full logs, run 'nix-store -l /nix/store/x4qi0gqrfw2sjzss8g73xf77z7kqscng-python3.9-geoip2-4.6.0.drv'.
```

### Additional context

Add any other context about the problem here.

### Notify maintainers

@SuperSandro2000
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.19.0-41-generic, Ubuntu, 22.04.2 LTS (Jammy Jellyfish), nobuild`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.15.0`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
```

--- a/pkgs/development/python-modules/geoip2/default.nix
+++ b/pkgs/development/python-modules/geoip2/default.nix
@@ -14,14 +14,14 @@
 
 buildPythonPackage rec {
   pname = "geoip2";
-  version = "4.6.0";
+  version = "4.7.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-8OgLzoCwa7OL0Iv0h31ahONU6TIJXmzPtNJ7tZj6T4M=";
+    hash = "sha256-O93kmU9ryRfq+rW1Hnctc3sq4AA3pbhQAfsG3Gj3ed8=";
   };
 
   propagatedBuildInputs = [

--- a/pkgs/development/python-modules/maxminddb/default.nix
+++ b/pkgs/development/python-modules/maxminddb/default.nix
@@ -11,12 +11,12 @@
 
 buildPythonPackage rec {
   pname = "maxminddb";
-  version = "2.2.0";
+  version = "2.3.0";
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "e37707ec4fab115804670e0fb7aedb4b57075a8b6f80052bdc648d3c005184e5";
+    sha256 = "sha256-Egkg3d2VXzKuSMIHxs72/V3Ih0qIm6lLDywfc27N8wg=";
   };
 
   buildInputs = [ libmaxminddb ];

